bin/improver-spot-extract

```diff
@@ -247,37 +239,37 @@ def main():
                 args.data_path, diagnostic_filepath))
         diagnostic_name = diagnostics[diagnostic_key]["diagnostic_name"]
         # Load cubes into an iris.cube.CubeList.
-        diagnostics[diagnostic_key]["data"] = Load('multi_file').process(
-            files_to_read, diagnostic_name)
+        diagnostics[diagnostic_key]["data"] = (
+            Load('multi_file').process(files_to_read))
```

Comment:
I think the reason for taking this out is because the name of the diagnostic within the file doesn't always match the `diagnostic_name`. This can be imagined for probabilistic cubes, where currently the input cube may have a name, such as `probability_of_air_temperature`, rather than `air_temperature`, so loading when constraining upon `diagnostic_name` will return no cubes.